packages/fx-core/templates/plugins/resource/identity/bicep/identityProvision.template.bicep

```diff
@@ -0,0 +1,12 @@
+param provisionParameters object
```

Comment:
also add `@secure()` decorator because it contains credentials

Reply:
It is fixed.